core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingModalService.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service which handles opening and closing the training modal
+ * used for bothunresolved answers and answers within the training data of a classifier.
```

Comment:
Please add a space between "both" and "unresolved".

Reply:
@shubha1593 , I have made up some mistakes while I was resolving conflict , I.e why I am getting some error in Travis tests , can I do the work again and make another pull request ?